components/shared/loanMaterialModal/LoanMaterialModal.tsx

```diff
@@ -63,7 +66,7 @@ const LoanMaterialModal = ({
   const { mutate } = usePostV1UserLoansIdentifier()
   const isbns = getIsbnsFromManifestation(manifestation)
   const [isHandlingLoan, setIsHandlingLoan] = useState(false)
-  const [error, setError] = useState<Error | null>(null)
+  const [publizonError, setPublizonError] = useState<{ code: number; message: string } | null>(null)
```

Comment:
Again to make it clear what a code is, I think it would make sense again to use the generated [ApiResponseCode](https://github.com/danskernesdigitalebibliotek/dpl-go/blob/5b777a0f5a7b0ef554c54cc503f9ba35dfb54d01/lib/rest/publizon/local-adapter/generated/model/apiResponseCode.ts#L16C14-L16C29).